feature-detects/scrolltooptions.js

```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "scrollToOptions dictionary",
+  "property": "scrolltooptions",
+  "caniuse": "mdn-api_scrolltooptions",
```

Comment:
I wonder if adding the mdn caniuse calls is of any utility 🤔. I mean, don't delete, it's okay, great job!
What I mean is that if we are not able to add them to the [main caniuse file](https://github.com/Modernizr/Modernizr/blob/master/test/browser/integration/caniuse.js) because they cause errors, what is the point adding them to the files?

Reply:
The caniuse file is there for testing, the tag here is for displaying it on the website. So having a mdn here is ok in my opinion.